gtest/Cargo.toml

```diff
@@ -14,7 +14,8 @@ codec = { package = "parity-scale-codec", version = "3.1.2", features = ["derive
 logger = { package = "log", version = "0.4.17" }
 hex = "0.4.3"
 colored = "2.0.0"
+derive_more = "0.99.17"
 env_logger = "0.9.0"
 path-clean = "0.1.0"
-wasm-instrument = { version = "0.1" }
+wasm-instrument = "0.1.1"
```

Comment:
we depend on "0.1" everywhere except `gear_core`

Reply:
You know me to love three-number versions 😄 But ok, reverted it back.